src/modules/translation/languages/en.json

```diff
@@ -611,16 +611,16 @@
   "land_assign_ens_page": {
     "title": "Assign {name}",
     "subtitle": "Assign a name to {land}",
-    "set_name_message": "In order to set this name to your parcel you first need to send 2 transactions.",
+    "set_name_message": "Before you can assign this name to your parcel, you need to submit 2 transactions: one that translates the human readable, alphanumeric name into an Ethereum address and one that creates a hash code linking that generated address to your LAND.",
     "set_resolver": "Set Resolver",
-    "set_resolver_explanation": "The Resolver is a Smart Contract responsible for the process of translating names into addresses.",
+    "set_resolver_explanation": "This translates your name into an Ethereum address.",
```

Comment:
```diff
- This translates your name into an Ethereum address.

+ This translates your name into an Ethereum address, IPFS hash, or other decentralized standard formats.
```

Reply:
same as above, what the resolver does in the end is mapping the string that represents your name into an ethereum address, aren’t we getting too technical on this?

Reply:
Hmmm `[...]is mapping the string that represents your name into an Ethereum address` this is only correct when your record is a Ethereum address type. You can have an IPFS link, a Swarn link, etc.

Thinking it for a basic user, and also, based on the comment above, maybe we can go with 

```
This allows your name to be linked to a parcel.
```

Reply:
Done! LGTM!